### PR TITLE
improve: more features in types tensor and numpy

### PR DESCRIPTION
**Pull request by samedii**

### CODE REVIEW

File: lantern/numpy.py [hunk 1]
```diff
@@ -1,34 +1,25 @@
...
         elif isinstance(data, np.ndarray):
             return data
+        elif isinstance(data, torch.Tensor):
+            return data.numpy()
         else:
             return np.array(data)
 
     @classmethod
-    def ndim(cls, ndim):
+    def ndim(cls, ndim) -> "Numpy":
```
Comment (Jim-Holmstroem) [inline lantern/numpy.py]: Since lantern is python 3.7+
```
from __future__ import annotations
```
and have it the "modern way" ` -> Numpy`

Comment (samedii) [reply lantern/numpy.py]:
```
NameError: name 'Numpy' is not defined
```

Comment (samedii) [reply lantern/numpy.py]: Nevermind, you are right. It works with the import :+1:

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -8,32 +8,28 @@ jobs:
...
-        poetry install
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: Install dependencies
+        run: |
+          curl -sSL https://install.python-poetry.org | python - --version 1.2.1
+          $HOME/.local/bin/poetry install
```
Comment (Jim-Holmstroem) [inline .github/workflows/publish.yml]: https://github.com/nextml-code/nextrabbit/blob/main/.github/workflows/test.yml#L15

**Comment by Jim-Holmstroem** [comment]
Awesome push :D

**Review by Jim-Holmstroem — APPROVED** [review]

**Review by FelixAbrahamsson — APPROVED** [review]

**Review by JoarGruneau — APPROVED** [review]

**Pull request merged**